swagger.yaml

```diff
@@ -39,23 +127,160 @@ parameters:
+                  description: The order this stop is used relative to the other stops
+  VehiclesResultObject:
+    properties:
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/VehicleResourceObject"
+  VehicleResultObject:
+    properties:
+      data:
+        $ref: "#/definitions/VehicleResourceObject"
+  VehicleResourceObject:
+    properties:
+      id:
+        type: string
+        description: vehicleID
+      type:
+        type: string
+        description: route
+      attributes:
+        properties:
+          routeID:
+            type: string
+            description: routeID for route of vehicle
+          name:
+            type: string
+            description: Name of vehicle
+          latitude:
+            type: number
+            format: float
+            description: Latitude of vehicle's position
+          longitude:
+            type: number
+            format: float
+            description: Longitude of vehicle's position
+          speed:
+            type: number
+            format: float
+            description: Speed of vehicle (MPH)
+          heading:
+            type: integer
+            description: Compass heading of vehicle (0-359)
+          age:
+            type: integer
+            description: Seconds since vehicle position was updated
+          onRoute:
+            type: boolean
+            description: Whether the vehicle is on its route
+          delayed:
+            type: boolean
+            description: Whether the vehicle is delayed
+  ArrivalsResultObject:
+    properties:
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/ArrivalResourceObject"
+  ArrivalResultObject:
+    properties:
+      data:
+        $ref: "#/definitions/ArrivalResourceObject"
+  ArrivalResourceObject:
+    properties:
+      id:
         type: string
-        description: Git commit hash of the revision which was built.
-      documentation:
+        description: arrivalID
+      type:
         type: string
-        description: Filename of the swagger specification for the API.
+        description: route
+      attributes:
+        properties:
+          routeID:
+            type: string
+            description: routeID for route of vehicle
+          stopID:
+            type: string
+            description: Name of vehicle
+          arrivals:
+            type: array
+            description: Times that this route is arriving at this stop
+            items:
+              properties:
+                eta:
+                  type: string
+                  format: date
+                  description: Time and date that route is expected to arrive at stop
+                secondsUntilArrival:
+                  type: integer
+                  description: Seconds until expected arrival
+                arriving:
+                  type: boolean
+                  description: Whether the vehicle is arriving
```

Comment:
What does this field mean? When wouldn't a vehicle be arriving?

Reply:
From what I can see from the vendor's API, arriving is `true` when a stop is the next one that a bus will be stopping at.